Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/NodeFactory.php

```diff
@@ -73,13 +75,18 @@ public function mapNodeRowToNode(
             : null;
 
         return Node::create(
+            NodeAddress::create(
```

Comment:
I really wonder why we keep the NodeFactory so "separate"? I guess we could remove the ContentRepositoryId it holds and just directly give a full NodeAddress to mapping from the ContentSubGraph? That feels neater.

Reply:
could you please elaborate? Do you want to add more parameters to the `NodeFactory`s constructor?

Reply:
Ok so idea would be:

The two map functions directly accept a NodeAddress instead of the separate values, and in turn the NodeFactory doesn't even need the contentrepositoryid as field anymore I think.

Reply:
True, kindof good idea ... but we need the node identifier for the `NodeAddress` but its in the $nodeRow thing and it would be weird to map that beforehand? 

Also for the multiple thing `mapNodeRowsToNodes` we have the same problem...